examples/ding-dong-bot-oop.py

```diff
@@ -63,12 +63,12 @@ async def on_message(self, msg: Message):
             
             # reply the image
             image_filebox = FileBox.from_file('./hd-image.jpg')
-            await community.say(image_filebox)
+            await room.say(image_filebox)
             # await msg.say(hd_file_box)
             
         elif msg_type==MessageType.MESSAGE_TYPE_URL:
             content = await msg.to_url_link()
```

Comment:
content variable is not clear for urllink data, which is casual. I suggest `url_link` variable to identify it. How do you think it?